bin/npm.Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM node:6.10.3
+RUN yarn global add npm@5 --no-progress
```

Comment:
We don't use yarn

Reply:
sorry about that. I really didn't wan to introduce an extra tool into the mix. 
yarn is only being used here to install npm 5 in the node image. Yarn proved to be the a simple and consistent way to install npm@5. Since it comes with the image, there's no need to install anything extra.

The whole thing is sort of a chicken-egg problem. I tried the steps at https://github.com/npm/npm#super-easy-install, but using an old npm to install a new npm is not reliable. I also tried many variations of the suggested `curl -L https://www.npmjs.com/install.sh | sh`. In addition to offering less control over the npm version being installed, it wasn't very reliable.

Thanks for reviewing this, by the way. And since I got your attention, let me ask you something. Which version of node are you using in production? The readme only says "node.js >= v6". Using this docker approach, you can pin it and guarantee a much more consistent run in CI or for anyone trying to contribute. I chose 6.10.3 as just a guess.

As a final note, let me say that I was the one that originally created this make+docker solution about 2 years ago. I built the integration with circle. At some point, it was dropped for some reason. If are interested, I could help fix that integration again. It was done in a private ProtonMail repo where the WebClient v3 was being built. In the copy of that here, the history got lost. You can ask @jasonstockman, though. I even chatted with them about the whole thing over slack at the time.

Reply:
```sh
$ npm -v && node -v    
5.4.0                  
v8.2.1                 
```

As we use node for the build process, you can use `nvm` to install both node and npm. I didn't have any issues to move from npm 3 to 5 via `npm i -g npm@latest`, but maybe it doesn't work everytime, idk.

ok I will ask thx

Reply:
the official node 8 images come with npm 5, removing the need to install it at all. I'll use that here.